internal/telemetry/segment.go

```diff
@@ -154,3 +154,13 @@ func timeFromUnixTimestamp(timestamp string) (time.Time, error) {
 func UnixTimestampFromTime(t time.Time) string {
 	return strconv.FormatInt(t.Unix(), 10)
 }
+
+func shellAccess() shellAccessKind {
+	if os.Getenv("START_WEB_TERMINAL") == "1" {
```

Comment:
OK, but ... Could you add a TODO/NOTE of some kind? At some point we'll have VMs with Gotty started that can be accessed via ssh *or* web, and this would treat them all as web. Really what we care about is the connection, that is running the command, not whether the VM has Gotty running or not.

Reply:
sure I can add a comment...

Reply:
If you look at how `START_WEB_TERMINAL` works, its purpose _is_ to ensure that we start the VM for browser shells. Whenever that logic changes, we'll need to update this as well.

@LucilleH suggested using this. Is there another more persistent way to detect that the shell is running as a browser shell?

Reply:
I think this is the right behavior for now – I'm just saying we'll need to improve it and I like adding a TODO so it doesn't slip our minds.

Reply:
The other approach is to rely on an SSH env var like `SSH_USER_PUBLIC_KEY` so:

```
// Check if in cloud
if os.Getenv("DEVBOX_REGION") != "" {
   if os.Getenv("SSH_USER_PUBLIC_KEY") != "" {
       return ssh
   } else {
       return browser
    }
}
return local
```

(need to test this)

Reply:
I can use `SSH_TTY`  

I dumped `env` for both ssh and browser shells to compare:
```
❯ cat web_env.txt | grep SSH
SSH_HOST_KEY=
DEFAULT_SSH_USER=devbox
SSH_USER=savil
SSH_USER_PUBLIC_KEY=
SSH_LISTEN_IPV4=0.0.0.0
SSH_LISTEN_IPV6=::
SSH_PORT=2222

❯ cat ssh_env.txt | grep SSH
SSH_CONNECTION=fdaa:0:a780:a7b:b2e2:41a9:b82f:2 58826 fdaa:0:a780:a7b:a1:ef3c:55b5:2 2222
SSH_CLIENT=fdaa:0:a780:a7b:b2e2:41a9:b82f:2 58826 2222
SSH_TTY=/dev/pts/0
```

Reply:
I think for now this is fine to use `START_WEB_TERMINAL` 👍